CMakeLists.txt

```diff
@@ -1425,3 +1425,19 @@ if(CONFIG_X86 AND CONFIG_USERSPACE AND NOT CONFIG_X86_NO_MELTDOWN)
       does not contain a fix for this issue."
   )
 endif()
+
+if(CMAKE_HOST_WIN32)
+  if(DEFINED CMAKE_C_COMPILER_VERSION AND (CMAKE_C_COMPILER_VERSION VERSION_EQUAL 8.2.1))
+	if(NOT DEFINED SUPPRESS_ISSUE_12257)
```

Comment:
We should start documenting and listing those special variables somewhere and possibly namespace them consistently to make it easy to find them across the build system. We are adding all kind of workaround managed by such variables already, so time to have some structure..

Reply:
I completely agree that the CMake variables that are user-settable should be listed in documentation.

Unfortunately I am unable to work on this at this time.

Reply:
Is the issue with the version of objcopy, not the version of GCC?